core/client/fs/src/main/java/alluxio/client/file/cache/store/RocksPageStore.java

```diff
@@ -81,17 +83,18 @@ public static RocksPageStore open(RocksPageStoreOptions options) throws IOExcept
       }
```

Comment:
close option too before `}` just to be complete--- but I understand it matter little here as we are throwing IOE

Reply:
nice catch